### ISSUE TITLE
Deprecate and remove `dump_profile_cookies`

### ISSUE THREAD

**Issue by englehardt** [mozilla/OpenWPM#259]
The `dump_profile_cookies` command saves a snapshot of Firefox's `cookies.sqlite` database. This was added before we had other instrumentation to monitor cookies. Now that we can monitor cookies via an extension, we should drop support for `dump_profile_cookies`. Monitoring within an extension is preferable because it provides more context for each cookie operation and we don't have to worry about pending writes from Firefox.

**Comment by englehardt** [comment]
In the upgrade to FF60+ we found that Firefox doesn't regularly commit to the `cookies.sqlite` database, and reads would fail with the error: `OperationalError: database is locked`. We tried using [`sleep_until_sqlite_checkpoint`](https://github.com/mozilla/OpenWPM/blob/1ab398b37bf1535eaf382a6970e78839bc7e6347/automation/Commands/utils/firefox_profile.py#L17), but found that it would wait longer than the timeout. The only way to read data was to copy the database and read from a copy (which wouldn't have the uncommitted data). Given these issues, the data read from the database is likely to be incomplete.
